scipy/stats/tests/test_sampling.py

```diff
@@ -806,21 +806,28 @@ def test_basic(self, dist, mv_ex):
         rng = NumericalInversePolynomial(dist, random_state=42)
         check_cont_samples(rng, dist, mv_ex)
 
-    very_slow_dists = ['studentized_range', 'trapezoid', 'triang', 'vonmises',
-                       'levy_stable', 'kappa4', 'ksone', 'kstwo', 'levy_l',
-                       'gausshyper', 'anglit']
-    # for these distributions, some assertions fail due to minor
-    # numerical differences. They can be avoided either by changing
-    # the seed or by increasing the u_resolution.
-    fail_dists = ['ncf', 'pareto', 'chi2', 'fatiguelife', 'halfgennorm',
-                  'gibrat', 'lognorm', 'ncx2', 't']
-
     @pytest.mark.xslow
     @pytest.mark.parametrize("distname, params", distcont)
     def test_basic_all_scipy_dists(self, distname, params):
-        if distname in self.very_slow_dists:
+
+        very_slow_dists = ['anglit', 'gausshyper', 'kappa4',
+                           'ksone', 'kstwo', 'levy_l',
+                           'levy_stable', 'studentized_range',
+                           'trapezoid', 'triang', 'vonmises']
```

Comment:
I moved these from above because they are only used in `test_basic_all_scipy_dists`.